lib/models/User.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:anxiety_go/models/difficulty.dart';
+
+class User {
+  final String firstName;
+  final String lastName;
+  final String email;
+  final String password;
+  final Difficulty difficultyLevel;
```

Comment:
I think you should remove the 'final' keyword from all the variables since the user should be able to change their profile information.